comm_norms.adoc

```diff
@@ -0,0 +1,460 @@
+In this section, we'll discuss using an issue tracker for the purpose of reporting failures with the software.
+By reporting your issue using a project's issue tracker, you ensure that maintainers who are looking out for problems will see your report and act upon it.
+
+=== Why file an issue?
+
+While filing an issue may seem more cumbersome than simply asking for help in real-time chat, it is important to do for several reasons:
+
+* Project contributors cannot keep track of all conversations occurring across various platforms, but they can always refer back to the issue tracker when they have an opportunity to work on improving the project.
+Real-time chat and social media are ephemeral communication channels; the issue tracker is a purpose built tool for recording and reviewing problems with the software.
+* Software projects often define their upcoming work plans by using their issue tracker as a key component—and perhaps their sole tool—to prioritize all possible areas to work on (simply put, the project's issue tracker is very often synonymous with the project's to-do list).
+If your problem does not make its way into the issue tracker, it will likely not be addressed simply because a very busy person has forgotten the details of the problem.
+For this reason, you will often find that one of the first requests you receive when asking for help is to file an issue so that the project maintainers can keep track of the problem.
+* Filing an issue allows you and the project contributors to communicate asynchronously about the problem in an easy way, as all parties can refer back to and access the issue description and follow up comments at any time.
+* When you've uncovered a problem with the software, you might discover that the problem is actually the root cause of _another_ problem, or there may be a way in which _several_ problems are related.
+Issue tracking software allows project developers to easily group related issues together, which may aid in diagnosing a problem's root cause.
+* People often encounter the same issues with software, and many of them are filing issues with the project.
+Having multiple reports of the same problem can be very time consuming for the project maintainers, as they would then need to respond to each individual reporter about work in progress.
+Fortunately, issue trackers make this process easier by allowing maintainers to quickly and easily close issues by stating that they're duplicates of an existing one (and then asking the reporter to track work-in-progress in the "original" report).
+Project maintainers can then engage in broadcast-style communication to everyone experiencing the problem in one place, simplifying their work stream while still helping everyone who needs assistance.
+
+=== Make finding it easy
```

Comment:
Something I notice as I work on adapting some of this material for downstream use: Until this point, the chapter addresses its advice on issue trackers to potential *contributors*. But this section is directed at community managers. The next session (on templates) also addresses community managers, but adapting its advice for an audience of contributors is a bit easier. Just something we may wish to consider in upstream editing.